src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import "./Introduce.scss";
+
+function Introduce() {
+  const navigate = useNavigate();
+
+  const goToProduct = () => {
+    navigate("/product");
+  };
+
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  return (
+    <>
+      <main className="IntroduceMain">
+        <div className="box">
+          <img alt="위코드로고" src="images/wecode.webp" />
+          <p className="name">조수진</p>
+          <p className="number">46기</p>
+          <div className="boxBottom">
+            <input onChange={handleInput}></input>
+            <button onClick={goToProduct}>클릭</button>
+          </div>
+          <Link className="pageToPage" to="/product">
+            product 페이지로 이동
+          </Link>
+        </div>
+
+        <div className="secondGround">
+          {INTRO_CARD.map((cardList) => {
+            return (
+              <div className="secondBox" key={cardList.key}>
+                <div className={`secondBackground ${cardList.name}`} />
+                <p className="secondName">이름: {cardList.name}</p>
+                <p className="rgn">rgb: {cardList.rgb}</p>
+              </div>
+            );
+          })}
+        </div>
+      </main>
+    </>
```

Comment:
```suggestion
      <main className="introduceMain">
        <div className="box">
          <img alt="위코드로고" src="images/wecode.webp" />
          <p className="name">조수진</p>
          <p className="number">46기</p>
          <div className="boxBottom">
            <input onChange={handleInput}></input>
            <button onClick={goToProduct}>클릭</button>
          </div>
          <Link className="pageToPage" to="/product">
            product 페이지로 이동
          </Link>
        </div>

        <div className="secondGround">
          {INTRO_CARD.map((cardList) => {
            return (
              <div className="secondBox" key={cardList.key}>
                <div className={`secondBackground ${cardList.name}`} />
                <p className="secondName">이름: {cardList.name}</p>
                <p className="rgn">rgb: {cardList.rgb}</p>
              </div>
            );
          })}
        </div>
      </main>
```
main 태그가 가장 부모 태그기 때문에 fragment 태그로 감싸지 않아도 될 것 같습니다!
그리고 가장 부모 태그의 className은 컴포넌트의 이름과 동일하게 작성하는 컨벤션은 잘 하셨지만
소문자로 시작해야 해요!